doc/developer/bestPractices/ContributorInfo.txt

```diff
@@ -13,6 +13,16 @@ account, signing the contributor agreement, and submitting pull requests.
 Developer Workflow
 ------------------
 
+Overview:
+
+#. `Fork repo on github`_
+#. `Create new branch`_
+#. `Submit pull request`_
+#. `Test changes`_ (can be done by team at Cray)
+#. `Mail chapel-developers@ for review/merge`_
+
+.. _Fork repo on github:
```

Comment:
I believe they work on HTML as well, it's just difficult to tell without the visual indicator that you've landed where you expect.